index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
it would be better if you add the full doctype declaration for html5 including the required meta for this project as per the requirement in the link here: [#](url)